python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -352,6 +356,55 @@ def __new__(
         )
 
 
+@whitelist_for_serdes
+@experimental
+class BackfillRequest(
+    NamedTuple(
+        "_BackfillRequest",
+        [
+            ("asset_partitions", "AssetGraphSubset"),
```

Comment:
confirming understanding - the reason AssetGraphSubset is not also in the public API is because you are not supposed to construct one directly?